src/hypermedia-constants.js

```diff
@@ -543,8 +553,16 @@ export const Actions = {
 		updatePreventMovingBackwards: 'update-prevent-moving-backwards',
 		updateAutoSetGraded: 'update-auto-set-graded',
 		delete: 'delete-quiz',
+		updateDescription: 'update-description',
 		updateAttemptsAllowed: 'update-attempts',
-		updateDescription: 'update-description'
+		timing: {
+			updateType: 'update-timing-type',
+			updateTimeLimit: 'update-timing-time-limit',
+			updateLateTypeId: 'update-timing-late-type-id',
+			updateHasTimer: 'update-timing-has-timer',
+			updateEnforcedTimeLimitGracePeriod: 'update-enforced-time-limit-grace-period',
```

Comment:
sorry, i meant it should be `update-timing-grace-limit` instead of `update-enforced-time-limit-grace-period`